packages/admin/admin/src/stack/useIsLastVisibleSwitch.ts

```diff
@@ -0,0 +1,15 @@
+import { useStackApi } from "./Api";
+import { useStackSwitchApi } from "./Switch";
+
+export const useIsLastVisibleSwitch = () => {
+    const stackApi = useStackApi();
+    const stackSwitchApi = useStackSwitchApi();
+
+    if (stackApi && stackSwitchApi) {
+        const currentSwitchIndex = stackSwitchApi.id ? stackApi.switches.findIndex((i) => i.id === stackSwitchApi.id) : -1;
+        const nextSwitchShowsInitialPage = stackApi.switches[currentSwitchIndex + 1] && stackApi.switches[currentSwitchIndex + 1].isInitialPageActive;
+        return currentSwitchIndex === stackApi.switches.length - (nextSwitchShowsInitialPage ? 2 : 1);
+    }
```

Comment:
We also have this in RouterTabs: https://github.com/vivid-planet/comet/blob/main/packages/admin/admin/src/tabs/RouterTabs.tsx#L116-L122

Could we use it here as well?

Reply:
I think so, I'll use the hook here as well 👍🏼